argocd/overlays/moc-infra/applications/envs/emea/morty/cluster-management/odf.yaml

```diff
@@ -0,0 +1,20 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: odf
+spec:
+  destination:
+    name: morty
+    namespace: argocd
```

Comment:
did you mean `namespace: openshift-storage` ?

Reply:
Indeed, fixed. Thanks!